src/stores/Form.js

```diff
@@ -1714,6 +1777,14 @@ class FormStore {
             }));
         });
 
+        // Searchables
+        let searchables = {};
+        toJS(localizedData.searchables).forEach(({id, versionHash}) => {
+          searchables[id] = {
+            "/": UrlJoin("/qfab", versionHash, "meta", "searchables")
```

Comment:
Use the `CreateLink` method. This is not setting the `auto_update` attribute, which is important.